### ISSUE TITLE
No Clangd

### ISSUE THREAD

**Issue by svenefftinge** [gitpod-io/workspace-images#108]
I opened a project and got

<img width="515" alt="Screenshot 2019-09-09 at 09 42 24" src="https://user-images.githubusercontent.com/372735/64512324-6cbac000-d2e6-11e9-8bc4-6d328206183f.png">

Also in workspace-full I don't see any `clangd` nor `clangd-9` on the `PATH` or in `/usr/bin`

**Comment by jankeromnes** [comment]
Wow, indeed. 😱

```
$ ls -larth /usr/bin/clangd*
lrwxrwxrwx 1 root root 17 Sep  3 13:31 /usr/bin/clangd -> /usr/bin/clangd-9
```

(You can't see it here, but the symbolic link is broken, because `/usr/bin/clangd-9` doesn't exist.)

Not sure how this could have happened, but I'll fix it.

**Comment by jankeromnes** [comment]
Fixed in b6526e913451e3c8d5cb3d949dfc8c276946df86.

It seems that we never explicitly installed `clangd`. I'm super confused as to how this even worked before.
